src/views/web/index.php

```diff
@@ -1,79 +1,94 @@
 <?php
-/**
- * @var Yiisoft\Requirements\RequirementsChecker $this
- * @var array $summary
- * @var array[] $requirements
- */
-?><!DOCTYPE html>
+/* @var $this Yiisoft\Requirements\RequirementsChecker */
```

Comment:
It's incorrect. See: https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/var.html

Previous is OK.